pkg/master/master.go

```diff
@@ -292,15 +293,20 @@ type nodeAddressProvider struct {
 	nodeClient coreclient.NodeInterface
 }
 
-func (n nodeAddressProvider) externalAddresses() (addresses []string, err error) {
+func (n nodeAddressProvider) addresses() (addresses []string, err error) {
+	preferredAddressTypes := []api.NodeAddressType{
+		api.NodeExternalIP,
+		api.NodeInternalIP,
+		api.NodeLegacyHostIP,
```

Comment:
would like someone familiar with the tunneling setup to comment... @mikedanese / @cjcullen? we weren't using NodeInternalIP for tunneling before... is that valid?